### PR TITLE
feat: change permission and role types

### PR DESCRIPTION
**Pull request by antsgar**

### CODE REVIEW

File: packages/auth/src/Domain/Role/Role.ts [hunk 1]
```diff
@@ -1,6 +1,13 @@
-export enum Role {
+import { Uuid } from '../Uuid/Uuid'
+
+export enum RoleName {
   User = 'USER',
```
Comment (karolsojko) [inline packages/auth/src/Domain/Role/Role.ts:4]: can we change this one to `BASIC_USER`?

File: packages/auth/src/Domain/Permission/Permission.ts [hunk 2]
```diff
@@ -1,5 +1,12 @@
-export enum Permission {
+import { Uuid } from '../Uuid/Uuid'
+
+export enum PermissionName {
   SyncItems = 'SYNC_ITEMS',
```
Comment (karolsojko) [inline packages/auth/src/Domain/Permission/Permission.ts:4]: could we change the permission names to the ones that are used? 
https://github.com/standardnotes/auth/blob/develop/migrations/1624434102642-update_roles_and_permissions.ts#L17-L46

**Review by karolsojko — CHANGES_REQUESTED** [review]

File: packages/auth/src/Domain/Permission/Permission.ts (hunk 2 above)
Comment (antsgar) [reply packages/auth/src/Domain/Permission/Permission.ts]: Done!

File: packages/auth/src/Domain/Role/Role.ts (hunk 1 above)
Comment (antsgar) [reply packages/auth/src/Domain/Role/Role.ts]: Done!

**Review by karolsojko — APPROVED** [review]

**Pull request merged**